### DISCUSSION TITLE
Chinese POS tag incorrect?

### DISCUSSION THREAD

**Discussion by bokailong** [explosion/spaCy#6885] [Language Support]
New to spaCy. I have been fooling with it and it seems great. But, my first time implementing a POS tag for Chinese identified the word red 紅色 as a VERB ROOT. I changed it to blue 蓝色 and it was identified correctly as ADJ ROOT. The sentence was My car is red 我的車是紅色的. I am using python 3.9 and spaCy 3.0 on linux.

**Comment by adrianeboyd** [comment]
Hi, the statistical models are not going be 100% correct, especially on data that's different from the training data, which for Chinese is mostly news/newspaper text. See the top post in #3052 for a longer explanation, and a similar issue related to colors is #4744.
